monai/networks/nets/swin_unetr.py

```diff
@@ -65,6 +65,7 @@ def __init__(
         use_checkpoint: bool = False,
         spatial_dims: int = 3,
         downsample="merging",
+        use_v2=False
```

Comment:
can we use the flag: version="v1" instead of use_v2=False?. We can use this flag to control different versions if there are variants in the future...

Reply:
Hi @wyli ,

I agree with @tangy5 , "use_v2" will cause breaking change if we add "v3" later.. an option "version: int = 1" may be better?
What do you think?

Thanks.

Reply:
this class will get complicated if we support three or more different modes, for readability I think we should start new classes for new architectural designs, like different versions of the inception models https://pytorch.org/vision/main/models/generated/torchvision.models.inception_v3.html